IntegrationTests/PurchaseTester/PurchaseTester.xcodeproj/project.pbxproj

```diff
@@ -26,6 +26,8 @@
 		3493336B21E69360007AAAB9 /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 3493336921E69360007AAAB9 /* LaunchScreen.storyboard */; };
 		3493337521E694D6007AAAB9 /* CatsViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3493337421E694D6007AAAB9 /* CatsViewController.swift */; };
 		34D05817233C1A3700343134 /* SwiftPaywall.swift in Sources */ = {isa = PBXBuildFile; fileRef = 34D05816233C1A3700343134 /* SwiftPaywall.swift */; };
+		5791CEC62743219A00E50C4B /* Constants.swift in Sources */ = {isa = PBXBuildFile; fileRef = 5791CEC52743219A00E50C4B /* Constants.swift */; };
```

Comment:
Double add in the project file

Reply:
It's in the main target and the watch extension :)

Reply:
I think? Let me double check.

Reply:
Ohhh that makes sense.

Reply:
Yup confirmed.